frontend/src/features/properties/list/columns.ts

```diff
@@ -39,7 +40,7 @@ export const columns: Column[] = [
   },
   {
     Header: 'Type',
-    accessor: '',
+    accessor: 'propertyTypeId',
```

Comment:
This field currently show as "0" or "1". Need to turn that into the appropriate Building or Land icon - next PR.